### PR TITLE
New ckanext-grouphierarchy version and personalised license list

### PR DESCRIPTION
**Pull request by MarijaKnezevic**

### CODE REVIEW

**Comment by MarijaKnezevic** [comment]
Changes are tested locally with `docer/compose` and with [sddi-ckan-k8s](https://github.com/tum-gis/sddi-ckan-k8s)

> helm install ckan sddi-ckan/sddi-ckan --set "ckan.image.tag=sddi-base-pr-31-43be262" --set "ckan.image.repository=ghcr.io/tum-gis/ckan-sddi-dev" -n ckan --create-namespace --atomic --wait --timeout 10m --values "https://raw.githubusercontent.com/tum-gis/sddi-ckan-k8s/main/examples/docker-desktop/values-local.yml"

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 For releases `< 1.0.0` minor version steps may indicate breaking changes too.
 
+## [1.1.3] - 2023-08-03
+
+### Fixed
+
+-  upstream bugfix from [ckanext-grouphierarchy-sddi](https://github.com/tum-gis/ckanext-grouphierarchy-sddi) extension tum-gis/ckanext-grouphierarchy-sddi#17
```
Comment (BWibo) [inline CHANGELOG.md:13]: Please describe shortly, what was fixed. Just letting people know, that this is an upstream bugfix is not helpful.

File: sddi-base/Dockerfile [hunk 2]
```diff
@@ -192,6 +192,7 @@ RUN set -ex && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_schemas = ckanext.scheming:ckan_dataset.yaml" && \
   ckan config-tool "${CKAN_INI}" "scheming.presets = ckanext.scheming:presets.json ckanext.repeating:presets.json ckanext.composite:presets.json" && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
+  ckan config-tool "${CKAN_INI}" "licenses_group_url = https://raw.githubusercontent.com/tum-gis/ckanext-grouphierarchy-sddi/main/ckanext/grouphierarchy/licenses_SDDI.json" &&\
```
Comment (BWibo) [inline sddi-base/Dockerfile:195]: - Missing space after `&&`
- Is setting this here really required? If this file is the default, it should be shipped with the extension (which is the case, according to the URL) and set as the default value (local path inside the container) in the extension. The user should only have to take action, if he needs to overwrite this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -159,6 +166,7 @@ for production environments.**
 ### Known issues
 
 [Unreleased]: https://github.com/tum-gis/ckan-docker/compare/1.1.2...HEAD
```
[the remark this answers is not in the document]
Comment (MarijaKnezevic) [reply CHANGELOG.md:169]: Do you mean this :

> https://github.com/tum-gis/ckan-docker/blob/devel-update/CHANGELOG.md?plain=1#L170 ?

It is already changed for v1.1.3

File: sddi-base/Dockerfile (hunk 2 above)
Comment (MarijaKnezevic) [reply sddi-base/Dockerfile]: This is an extended license list  which is required for SDDI. With CKAN installation it comes [default licenses list](https://docs.ckan.org/en/2.9/maintaining/configuration.html#licenses-group-url).
With the URL which is defined in mentioned [Dockerfile](https://github.com/tum-gis/ckan-docker/blob/devel-update/sddi-base/Dockerfile#L195), I want to overwrite CKAN Default value and use customized list.

This value could be defined as ENV var which I've added in our [docker/compose](https://github.com/tum-gis/ckan-docker/blob/feature/compose/compose/2.9/ckan.env#L23-L24) installation draft.

**Review by BWibo — APPROVED** [review]

**Pull request merged**